setup.py

```diff
@@ -64,12 +64,12 @@
     'arrow>=0.17.0',
     'Faker>=2.0.3',
     'ftfy>=4.4.3,<5.0.0',
-    'idutils>=1.1.7',
```

Comment:
comes from marshmallow-utils, which now is required here to get the new `IdentifierSet` and `IdentifierScheme`